app-backend/datamodel/src/main/scala/com/azavea/rf/datamodel/AOI.scala

```diff
@@ -0,0 +1,48 @@
+package com.azavea.rf.datamodel
+
+import java.sql.Timestamp
+import java.util.{Date, UUID}
+
+import geotrellis.slick.Projected
+import geotrellis.vector.MultiPolygon
+import io.circe.Json
+
+// --- //
+
+/** A Project's Area of Interest.
+  * This represents an area on the map (a `MultiPolygon`) which a user has
+  * set filters for.  If a new Scene entering the system passes these filters,
+  * the Scene will be added to the user's Project in a "pending" state. If the
+  * user then accepts a "pending" Scene, it will be added to their project.
+  */
+case class AOI(
+  /* Database fields */
+  id: UUID,
+  createdAt: Timestamp,
+  modifiedAt: Timestamp,
+  organizationId: UUID,
+  createdBy: String,
+  modifiedBy: String,
+
+  /* Unique fields */
+  area: Projected[MultiPolygon],
+  filters: Json
```

Comment:
My inclination is `case class` because if we can somehow re-use the query parameters case class used [here](https://github.com/azavea/raster-foundry/blob/develop/app-backend/database/src/main/scala/com/azavea/rf/database/tables/Projects.scala#L392) the task of adding scenes to a project based on AoI could be straightforward.